proto/api/v1/workflow.proto

```diff
@@ -32,6 +32,11 @@ message ExecuteWorkflowRequest {
   string visibility = 5;
   // If true, start the workflow but do not wait for the status to be returned.
   bool async = 6;
+  // These env vars will be applied to each action that is run. If there is a
+  // conflict between an env var set here and in buildbuddy.yaml, these
+  // overrides will take precedence. Otherwise all env vars set in
+  // buildbuddy.yaml will still apply.
+  map<string, string> env_var_overrides = 7;
```

Comment:
Done - the only one I left was in ci_runner/main.go, because I thought it may be unclear the precedence between env vars set via flag vs on the command vs in buildbuddy.yaml